packages/authenticator/amplify_authenticator/lib/src/blocs/auth/auth_event.dart

```diff
@@ -58,7 +58,7 @@ class AuthUpdatePassword extends AuthEvent {
 }
 
 class AuthConfirmSignIn extends AuthEvent {
-  const AuthConfirmSignIn(this.data, {required this.rememberDevice});
+  const AuthConfirmSignIn(this.data, {this.rememberDevice = false});
```

Comment:
Can you remove `rememberDevice`? It doesn't seem to be used anywhere

Reply:
Looks like it's being used [in the bloc](https://github.com/aws-amplify/amplify-flutter/blob/main/packages/authenticator/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart#L116) & [on other sign in flows](https://github.com/aws-amplify/amplify-flutter/blob/2e3daaf19516cf6dda24583d536466e1652c3d6e/packages/authenticator/amplify_authenticator/lib/src/state/authenticator_state.dart#L347).

It could be refactored to just use the state value, but that's out of scope of this PR imo.